#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-skipdirs.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Exclude debian directory from subdirs.

@DPATCH@

diff -Naur traceroute-2.0.8.orig/Make.rules traceroute-2.0.8/Make.rules
--- traceroute-2.0.8.orig/Make.rules	2007-08-24 16:09:21.000000000 +0000
+++ traceroute-2.0.8/Make.rules	2007-08-25 07:28:03.000000000 +0000
@@ -63,7 +63,7 @@
 
 
 #  layout stuff
-SKIPDIRS = tmp%
+SKIPDIRS = tmp% debian
 INCLUDEDIRS = include%
 LIBDIRS = lib%
 MODDIRS = mod%
